modules/core/shared/src/main/scala/coursier/core/Resolution.scala

```diff
@@ -907,6 +853,7 @@ final class Resolution private (
     errorCache: Map[Resolution.ModuleVersion, Seq[String]] = errorCache,
     finalDependenciesCache: Map[Dependency, Seq[Dependency]] = finalDependenciesCache,
     filter: Option[Dependency => Boolean] = filter,
+    reconciliation: Option[Module => Reconciliation] = reconciliation,
```

Comment:
No need to add that one here (that method is only there for compatibility with the original `copy` method, it doesn't need to accept new parameters).

Reply:
So the related mima rules aren't needed either.

Reply:
Done